modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc

```diff
@@ -0,0 +1,108 @@
+
+*Tomas comment: In order to create an API you will need to have at least Exchange Contributor for the Business group where you are creating the asset and Manage APIs configuration for the environment where you are creating the API Instance. But keep in mind that that is different from the permission Ensure that you obtain the environment ID and organization ID. For the latter permission is just access to APIM.*
+
+To create an API using Anypoint Service Mesh, perform the following steps:
+
+. Using the Anypoint Service Mesh, type the following command to create the API:
++
+[source,text,linenums]
+----
+./service-mesh api create  
+--name=<API resource name> \
+--namespace=<namespace name> \
+--environmentId=<environment ID> \
+--groupId=<group ID of the asset, for example, organization ID> \
+--assetId=<asset ID on Exchange> \
+--version=<Asset version of the instance on exchange> \
+--instanceLabel=<ID of the instance on API Manager> \ 
+--specTags=<list of comma separated tags to be added to the asset on Exchange> \
+--instanceTags=<list of comma separated tags to be added to the instance on API Manager> \
+--user=<username of the Anypoint Platform user, creating the API instance> \
+--password=<password of the Anypoint Platform user, creating the API instance>
+----
++
+[%header%autowidth.spread]
+|===
+| Parameter Name | Description | Required/Optional | Default Value
+| `name` | The name of the API that you want to provision, for example, <orders-api>. | Required | N/A
+| `namespace` | The namespace in which you want to create the adapter, for example, “mulesoft-apps”. | Optional | default
+| `environmentId` | The ID of the environment where you want to create your API instance. | Required | N/A
+| `groupId` | The ID of the group of assets on Exchange, for example, the organization ID. You can find the organization ID on API Manager landing page or in Access Management. | Optional | Organization ID of the organization that contains the environment. 
+| `assetId` | The ID of the assets that are a part of an organization. Supports alphanumeric characters without spaces. | Required | N/A
+| `version` | The version of the asset on Exchange. *Needs to be semver* what is this chirag? | Required | N/A
+| `instanceLabel` | The ID of the API instances of an Asset within a given environment. Supports alphanumeric characters without spaces. | Required | N/A
+| `specTags` | Tags to be added to the asset on Exchange. |  Optional | N/A
+| `instanceTags` | Tags to be added to the API instance on API Manager. |  Optional | N/A
+| `user` | The username credential of the Anypoint Platform user, who is creating the API. | Required | N/A
+| `password` | The password of the Anypoint Platform user, who is creating the API. | Required | N/A
+|===
++
+. Verify the <<provision-adapter-configure-service-mesh-CLI#adapter_status, adapter status>>:
++
+`./service-mesh adapter list`
```

Comment:
@priyaayyar @tbruno Why is the adapter list section under api creation section? This is wrong and it needs to change to api list with this command 

>  ./service-mesh api list

Reply:
Done! Thanks!